### PR TITLE
[no ticket][risk=no] Puppeteer fix deleteWorkspace func failure

### PR DESCRIPTION
**Pull request by aweng98**
`WorkspaceCard.deleteWorkspace()` function could fail if delete took longer than 60 seconds. Failed in `workspace-duplicate-cdr-versions.spec.ts` test [here](https://app.circleci.com/pipelines/github/all-of-us/workbench/11495/workflows/d80070df-4913-4f55-886b-846a4f22b1b1/jobs/148187/tests) in CircleCI.

Test failure stacktrace

```
test name: OWNER can duplicate workspace to a newer CDR Version via Workspace card
status: failed
failure: [
  'TimeoutError: waiting for XPath `.//*[child::*[@data-test-id="workspace-card"]]//*[@data-test-id="workspace-card-name" and normalize-space(text())="aoutest-404061625158614"]` to be hidden failed: timeout 60000ms exceeded\n' +
    '    at new WaitTask (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:505:34)\n' +
    '    at DOMWorld.waitForXPath (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/DOMWorld.js:441:26)\n' +
    '    at Frame.waitForXPath (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/FrameManager.js:866:51)\n' +
    '    at Page.waitForXPath (/home/circleci/workbench/e2e/node_modules/puppeteer/lib/cjs/puppeteer/common/Page.js:1288:33)\n' +
    '    at Function.waitUntilGone (/home/circleci/workbench/e2e/app/component/workspace-card.ts:97:16)\n' +
    '    at Function.deleteWorkspace (/home/circleci/workbench/e2e/app/component/workspace-card.ts:36:25)\n' +
    '    at Object.<anonymous> (/home/circleci/workbench/e2e/tests/workspace/workspace-duplicate-cdr-versions.spec.ts:93:5)',
  'Error: Error: Protocol error (Runtime.callFunctionOn): Target closed.\n' +
    '    at /home/circleci/workbench/e2e/utils/waits-utils.ts:403:11\n' +
    '    at runMicrotasks (<anonymous>)\n' +
    '    at runNextTicks (internal/process/task_queues.js:62:5)\n' +
    '    at listOnTimeout (internal/timers.js:523:9)\n' +
    '    at processTimers (internal/timers.js:497:7)\n' +
    '    at Object.waitWhileLoading (/home/circleci/workbench/e2e/utils/waits-utils.ts:390:3)\n' +
    '    at WorkspaceEditPage.dismissDeleteWorkspaceModal (/home/circleci/workbench/e2e/app/page/workspace-base.ts:262:5)'
]
```

### CODE REVIEW

File: e2e/tests/workspace/workspace-share-modal.spec.ts [hunk 1]
```diff
@@ -8,8 +8,16 @@ import { makeWorkspaceName } from 'utils/str-utils';
...
+    {
+      accessRole: WorkspaceAccessLevel.Writer,
+      userEmail: config.writerUserName,
+      userAccessTokenFilename: config.writerUserAccessTokenFilename
+    },
+    {
+      accessRole: WorkspaceAccessLevel.Reader,
+      userEmail: config.readerUserName,
+      userAccessTokenFilename: config.readerUserAccessTokenFilename
+    }
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-share-modal.spec.ts:20]: `lint --fix` updated format.

**Comment by aweng98** [comment]
PR testing had one test failure. Failure was [RW-6885](https://precisionmedicineinitiative.atlassian.net/browse/RW-6885). Unrelated to this PR changes.

**Review by petesantos — APPROVED** [review]
Looks good. I think the delete timeout is an actual bug.

If we want to improve code quality and e2e testing visibility, this may be a good example of where to start. It can serve as a concrete prototype of treating these things as bugs the e2e tests caught that the team needs to deal with rather than "test flakiness".

A UI unit test wouldn't catch this, and an API unit test wouldn't catch these delays. Every time we tailor the e2e tests in such a way they always pass, the e2e tests lose value, the code quality is reduced and the users start getting an inferior product.

**Review by aweng98 — COMMENTED** [review]
Thank you.

**Pull request merged**